caravel/templates/manage_listings.html

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.html" %}
+{% block title %} Manage Listings {% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block content %}
```

Comment:
Ah I was thinking more a manage_listings route that renders index with args. So like:
```Python
@app.route('/manage_listings')
def manage_listings():
    # foo
    return render_template('index.html', type='manage listings')
````